### ISSUE TITLE
v2.3 wrap up issue

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#534]
The final bug for v2.3 release.
- [x] increment version numbers
- [x] run packaging processes
- [x] create labels and update branches
- [x] register Bower package
- [x] regenerate PNG images with post-processing compression (issue #44)?
- [x] testing
- [x] update documentation
- [x] update download page on mathjax.org
